common.tf

```diff
@@ -227,6 +235,7 @@ data "ignition_file" "sysctl_kernel_vars" {
 fs.inotify.max_user_watches=1048576
 fs.inotify.max_user_instances=8192
 vm.max_map_count=262144
+user.max_user_namespaces=0
```

Comment:
Sorry about being picky - should this be a separate commit so its not reverted accidentally?

We should cherry-pick from the release-v1.21 branch?

Reply:
yes, definitely. I will remove it and rebase to clean it from history. Then we can cherry pick. I was working in parallel with HH and threw this in here to avoid reverting (or waiting :))

Reply:
done